ignite3/src/main/java/site/ycsb/db/ignite3/IgniteAbstractClient.java

```diff
@@ -163,6 +176,14 @@ public void init() throws DBException {
         } else {
           initIgniteClientNode();
         }
+
+        DataStreamerOptions dsOptions = DataStreamerOptions.builder()
```

Comment:
Why not to move all DataStreamer-related objects and logic to `IgniteStreamerClient` so it would be closer to the usage and will not pollute other benchmarks with unused objects? You can override the `init()` method like it is done in `IgniteSqlClient.java` for example. :) 

The exception is the `RecordView` object. IMHO it's OK to leave it in this class.

Reply:
moved